repository/src/main/java/org/alfresco/repo/content/AbstractRoutingContentStore.java

```diff
@@ -416,4 +417,11 @@ public boolean delete(String contentUrl) throws ContentIOException
         }
         return deleted;
     }
+
+    @Override
+    public boolean isStorageClassesSupported(Set<String> storageClasses)
+    {
+        // TODO: what should we do here? check whether at least one supports it similar to isContentUrlSupported?

```

Comment:
Yes, it should return true if at least one store supports it, similar to `isContentUrlSupported`.